src/components/Auth/AuthStyles.js

```diff
@@ -2,6 +2,13 @@ import styled, { css } from 'styled-components';
 import _PasswordField from 'material-ui-password-field';
 import _OutlinedInput from '@material-ui/core/OutlinedInput';
 import _Button from '@material-ui/core/Button';
+import _FormControl from '@material-ui/core/FormControl';
+
+export const ForgotTextContainer = styled.div`
```

Comment:
Not needed.

Reply:
If it is not applied.
![Capture](https://user-images.githubusercontent.com/32234926/60387853-624e9f80-9ac7-11e9-877a-4a492e13a5a3.JPG)

Reply:
@samagragupta No need for going till 320px, we are focussing on newer phones, hence till 375px max.

Reply:
@fragm3 @samagragupta  360px+ should be focussed.

Reply:
@akshatnitd @fragm3 Removed that